bridge/src/connection.rs

```diff
@@ -79,7 +82,15 @@ macro_rules! rpc_call {
     };
 }
 
-async fn make_rpc_call(mut client: Client, params: RpcParams) -> RpcResult {
+async fn make_rpc_call(client: Client, params: RpcParams) -> RpcResult {
+    match params.service.as_str() {
+        "workflow_service" => make_workflow_service_rpc(client, params).await,
+        "test_service" => make_test_service_rpc(client, params).await,
```

Comment:
At some point soon (maybe not now), you'll want to add `operator_service` here too

Reply:
Yes, good point, added a ticket to make sure it's not forgotten — https://github.com/temporalio/sdk-ruby/issues/112